src/StreamJsonRpc/Reflection/RpcMarshalableAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace StreamJsonRpc;
+
+using System;
+
+/// <summary>
+/// Marks an interface as marshalable.
```

Comment:
I feel like this could be beefed up a bit. How about:

> Designates an interface that is used in an RPC contract to marshal the object so the receiver can invoke remote methods on it instead of serializing the object to send its data to the remote end.

Then a `<remarks>` section that has something like this would probably be helpful:

```cs
/// <remarks>
/// <see href="https://github.com/microsoft/vs-streamjsonrpc/blob/main/doc/rpc_marshalable_objects.md">Learn more about marshable interfaces</see>.
/// </remarks>
```